frame/chainrelay/eth/relay/src/lib.rs

```diff
@@ -325,9 +377,10 @@ impl<T: Trait> Module<T> {
 
 			// If the new header has a lower number than the previous header, we need to cleaning
 			// it going forward.
-			// FIXME check overflow
 			if best_header_info.number > header_info.number {
-				for number in header_info.number + 1..=best_header_info.number {
+				for number in
```

Comment:
It's ok to use check_add here, but can also omit overflow check after carefully review.(Can't overflow here)